content/en/docs/tasks/administer-cluster/configure-upgrade-etcd.md

```diff
@@ -102,7 +102,7 @@ After configuring secure communication, restrict the access of etcd cluster to o
 
 For example, consider key pairs `k8sclient.key` and `k8sclient.cert` that are trusted by the CA `etcd.ca`. When etcd is configured with `--client-cert-auth` along with TLS, it verifies the certificates from clients by using system CAs or the CA passed in by `--trusted-ca-file` flag. Specifying flags `--client-cert-auth=true` and `--trusted-ca-file=etcd.ca` will restrict the access to clients with the certificate `k8sclient.cert`.
 
-Once etcd is configured correctly, only clients with valid certificates can access it. To give Kubernetes API server the access, configure it with the flags `--etcd-certfile=k8sclient.cert`,`--etcd-keyfile=k8sclient.key` and `--etcd-cafile=ca.cert`.
+Once etcd is configured correctly, only clients with valid certificates can access it. To give Kubernetes API server the access, configure it with the flags `--cert-file=k8sclient.cert`,`--key-file=k8sclient.key` and `--ca-file=ca.cert`.
```

Comment:
These are not `etcdctl` flags; they are flags for [`kube-apiserver`](https://kubernetes.io/docs/reference/command-line-tools-reference/kube-apiserver/)

I think the existing document is correct.